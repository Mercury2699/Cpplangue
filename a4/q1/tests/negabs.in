1 -2 NEG + x - ABS 2 * done
eval

set x 9
print eval

set y 7
print eval
