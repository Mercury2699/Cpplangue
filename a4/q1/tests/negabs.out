(|((1 + --2) - x)| * 2)
x has no value.
(|((1 + --2) - 9)| * 2)
12
(|((1 + --2) - 9)| * 2)
12
